### PR TITLE
Dmbm 742/review send special category data

### PR DESCRIPTION
**Pull request by gabimartini**

### CODE REVIEW

**Review by kjoshi — CHANGES_REQUESTED** [review]
Thanks Gabi,
Here are a few issues I've spotted on the Check page:

## Item titles
The name of each item doesn't match what's on the prototype. E.g.
image
The steps in the formdata all have names which are used on the overview pages. E.g.:
https://github.com/co-cddo/data-marketplace/blob/main/src/models/shareRequestTemplate.json#L155
It'd be good if these names could be re-used on the check page.

## Missing Not Required items
The items that are `Not Required` still need to be on the check page:
image
So I think we also need to include the `NOT REQUIRED` items in the loop and just display a _Not Required_ message.

## Large gaps
There are some quite large gaps at before quite a lot of the items. E.g.:
image
It'd be good to try and get rid of the gaps so it looks more similar to the prototype

## Project aims
The project aims seem to be displayed twice.
image

## Date
If no date is entered it is rendered on the page as `null/null/null`. It would be better to just have `Unrequested`, like what will be shown on the [Created Requests](https://cddodatamarketplace.atlassian.net/jira/software/projects/DMBM/boards/6?selectedIssue=DMBM-733) page
image

## Missing additional information
In the `Data travel outside UK` section if the answer is yes we also need to display the countries listed in the `data-travel-location` page.
image

The same also needs to be done in the `Other organisations accessing data` section
image

File: src/views/acquirer/check.njk [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% extends "page.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
```
Comment (AShaww) [inline src/views/acquirer/check.njk:8]: can we change this to full-width

File: src/helperFunctions/checkhelper.ts [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    "data-travel": {
+      data: {
+        no: {res:"No", attach: ""},
+        yes: {res: "Yes", attach: "data-travel-location", title: "Countries:"}
+    },
+      type: "list"
+  },
+    role: {
+      data:{
+        controller: "Controler",
```
Comment (AShaww) [inline src/helperFunctions/checkhelper.ts:158]: Can you take a look at this line please

**Review by AShaww — DISMISSED** [review]
This is looking really good Gabi.

I've requested a few changes.

Would it be possible to also implement something like this for the `date` if null and if lawful-personal/lawful-special is not required: 

image

Many thanks

**Comment by gabimartini** [comment]
> This is looking really good Gabi.
> 
> I've requested a few changes.
> 
> Would it be possible to also implement something like this for the `date` if null and if lawful-personal/lawful-special is not required:
> 
> image
> 
> Many thanks

Hi Amir, I've done your suggestions :) regard to this date and not required, this is requirement for the next ticket

**Review by kjoshi — APPROVED** [review]
All the remaining small issues were solved by #61, so this is good to go 👍

**Review by kjoshi — APPROVED** [review]
Small merge conflict needed sorting out. All good 👍

**Pull request merged**